week2/exercise P1008 in LuoGu.cpp

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+
+int isvalid(int Double, int Triple, int Flag);

```

Comment:
Sorry for misleading. Use `is_valid` instead. By the way, is there a specific reason that you use capitalized parameter name?

```suggestion
int is_valid(int _double, int triple, int flag);
```